src/test/java/seedu/sellsavvy/model/order/ItemTest.java

```diff
@@ -27,15 +27,15 @@ public void isValidItem() {
         // invalid item
         assertFalse(Item.isValidItem("")); // empty string
         assertFalse(Item.isValidItem(" ")); // spaces only
-        assertFalse(Item.isValidItem("^")); // only non-alphanumeric characters
-        assertFalse(Item.isValidItem("peter*")); // contains non-alphanumeric characters
```

Comment:
Is there a way to test for any non-printable characters (or characters that are not suitable) for the item name?